library/psa_crypto_its_internal.h

```diff
@@ -0,0 +1,149 @@
+/** \file psa_crypto_its_internal.h
```

Comment:
Note: github shows `psa_crypto_its_internal.h` being created, but not `psa_crypto_internal.h` being removed (or renamed to the other). That doesn't match the commit message.

Reply:
Fixed.